pkg/volume/gce_pd/gce_pd.go

```diff
@@ -55,6 +57,19 @@ const (
 	gceVolumeLimitKey           = "storage-limits-gce-pd"
 )
 
+// The constants and DiskAttachLimit are used to map from the machine type (number of CPUs) to the limit of
+// persistant disks that can be attached to an instance. Please refer to gcloud doc
+// https://cloud.google.com/compute/docs/disks/#increased_persistent_disk_limits
+const (
+	Default = iota
+	OneCpu
+	TwotoFourCPUs
+	EightCPUsandAbove
+	EightCPUs = 8
```

Comment:
It's kind of strange to mix cpu count constants with the DiskAttachLimit array index enum.

Reply:
Agree, but based on the doc https://cloud.google.com/compute/docs/disks/, the disk limits are directly related to number of CPUs.

Reply:
I mean can we keep two different const types? One to use as an enum for the array index, and one to define # of cpu constants.